services/cards-consultation/src/main/java/org/opfab/cards/consultation/repositories/UserUtilitiesCommonToCardRepository.java

```diff
@@ -58,7 +58,11 @@ default Flux<T> findByInitialParentCardUid(ReactiveMongoTemplate template, Strin
     default List<Criteria> computeCriteriaToFindCardByIdWithUser(String id, CurrentUserWithPerimeters currentUserWithPerimeters) {
         List<Criteria> criteria = new ArrayList<>();
         criteria.add(Criteria.where("_id").is(id));
-        criteria.add(computeCriteriaForUser(currentUserWithPerimeters));
+
+        boolean isCurrentUserMemberOfAdminGroup = ((currentUserWithPerimeters.getUserData().getGroups() != null) &&
+                                                   (currentUserWithPerimeters.getUserData().getGroups().contains("ADMIN")));
```

Comment:
i would factorize this method as we need it at least in two places 
adding the method inside CurrentUserWithPerimeters class 
and maybe a unit test

Reply:
After discussion not needed yet to factorize